test/index.test.js

```diff
@@ -1,3 +1,32 @@
-test('Initial test', () => {
-  expect(1).toBe(1);
+const request = require('supertest');
+const connection = require('../server/database/config/connection');
+const dbBuild = require('../server/database/config/build');
+
+const app = require('../server/app');
+
+beforeAll(() => dbBuild());
+
+afterAll(() => connection.end());
+
+test('Post cohort route', (done) => {
+  request(app)
+    .post('/api/v1/cohorts')
+    .send({
+      cName: 'G6',
+      cDescription: 'GazaSkyGeeks Code Academy, 6th Cohort',
+      cImgUrl: 'https://avatars0.githubusercontent.com/u/59821022?s=200&v=4',
+      cGithub: 'https://github.com/GSG-G8',
+    })
+    .expect(200)
+    .expect('Content-Type', /json/)
+    .end(async (err, res) => {
+      const { message } = res.body.data;
+      if (err) return done(err);
+      const result = await connection.query(
```

Comment:
since you are only using rows, you can destructure it into `{ rows } = await ...`